pages/Documents/ConversationalAI/Third-PartyBots/e-bot7/basic-content.md

```diff
@@ -0,0 +1,121 @@
+---
+pagename: Basic Content
+sitesection: Documents
+categoryname: "Conversational AI"
+documentname: Third-Party Bots
+subfoldername: e-bot7
+permalink: third-party-bots-e-bot7-basic-content.html
+indicator:
+---
+
+### The Welcome Event
+
+The behavior of the welcome event is different depending on whether the bot is for chat and messaging. This divergence comes down to the way that each individual LivePerson product works.
+
+#### Messaging Conversations
+
+A Messaging conversation qualifies as "initiated" from a Conversational Cloud perspective only after the consumer sends
+their first message. The consumer is prompted for their initial message in the channel they have chosen to initiate the
+conversation. As a result, the consumer’s first message will be the entry point for the main flow in e-bot7, which can be processed by a `Visitor Input` node.
+In this scenario, all bot messages in the flow which are created before this node will be skipped.
+
+<img class="fancyimage" style="width:450px" src="img/ThirdPartyBots/e-bot7-messaging-flow.png">
```

Comment:
@itay1313, did we give any guidance in the past around using HTML markup in Markdown files? I’d be inclined to use Markdown only, but this here is not the first time we do this. We could make an exception here for the images and figures, but should clean up later, and set up a policy for this.

Reply:
good question, I agree with you